lib/messages.ts

```diff
@@ -0,0 +1,84 @@
+import { expect, Page } from '@playwright/test';
+
+export enum Limit {
+  ten = 10,
+  twenty = 20,
+  fifty = 50
+}
+
+export enum FilterGroup {
+  offset = 'Offset',
+  timestamp = 'Timestamp',
+  epoch = 'Epoch timestamp',
+  latest = 'Latest messages'
+}
+
+export const setPartition = async function (page: Page, partition: string) {
+  await page.locator('[aria-label="Specify partition value"]').click();
+  await page.locator('[aria-label="Specify partition value"]').fill(partition);
+};
+
+export const pickFilterOption = async function (page: Page, option: FilterGroup) {
+  await page.locator('[data-testid="filter-group"]').click();
+  switch (option) {
```

Comment:
If I read the code correctly this `switch` can be replaced by:

```ts
await page.locator('a:text-is("' + option + '")').click();
```
or am I missing invariants?

Reply:
Correct, much more better without switch.